space/space.go

```diff
@@ -13,6 +13,8 @@ import (
 	"strings"
 )
 
+const SystemSpace = "system.space"
```

Comment:
@kwk so we insert this space into the table spaces to be used when doing a migration. For instance, we could refer to this space for those elements (wilt, wit, wi) that don't belong to any space. Does it make sense ?